lib/utils.js

```diff
@@ -27,8 +27,8 @@ export function storeToSparql(store) {
  * @returns {String} The triples in Turtle form. (No graphs, but it uses
  * prefixes at the top to make the file more human readable.)
  */
-export function storeToTtl(store) {
-  return storeToString(store, true);
+export async function storeToTtl(store) {
+  return await storeToString(store, false);
```

Comment:
not sure if this should be false here, but the `forSparql` parameter seemed to imply it. On further investigations this toggles between ntriples and turtle, so perhaps we just want ntriples here as well. In that case, just change it back to true
```suggestion
  return await storeToString(store, true);
```

Reply:
this has to be true indeed, we should use ntriple everywhere.